src/tweenable.rs

```diff
@@ -512,8 +541,9 @@ impl<T> Sequence<T> {
     #[must_use]
     pub fn from_single(tween: impl Tweenable<T> + Send + Sync + 'static) -> Self {
         let duration = tween.duration();
+        let boxed: BoxedTweenable<T> = Box::new(tween);
```

Comment:
It's probably safer to use
```suggestion
        let boxed: BoxedTweenable<T> = tween.into();
```
here so that users are not tempted to implement `Tweenable<T> for BoxedTweenable`.